packages/@coorpacademy-components/src/molecule/expandible-actionable-table/index.tsx

```diff
@@ -0,0 +1,220 @@
+import React, {useState} from 'react';
+import classnames from 'classnames';
+import {get, isString} from 'lodash/fp';
+import ButtonLinkIconOnly from '../../atom/button-link-icon-only';
+import StatusItem from '../../atom/status-item';
+import BulkProgressBar from '../bulk-progress-bar';
+import {WebContextValues} from '../../atom/provider/web-context';
+import Provider, {GetTranslateFromContext} from '../../atom/provider';
+import BulletPointMenuButton from '../bullet-point-menu-button';
+import ButtonLink from '../../atom/button-link';
+import ErrorsTable from '../errors-table';
+import style from './style.css';
+import {ExpandState, Field, LastField, NestedRow, Props, propTypes} from './types';
+
+const buildField = (field: Field) => {
+  if (isString(field)) return field;
+  const {componentType} = field;
+  switch (componentType) {
+    case 'status':
+      return <StatusItem {...field} />;
+    case 'progress-bar':
+      return <BulkProgressBar {...field} />;
+  }
+};
+
+const buildLastField = (lastField: LastField) => {
+  const {componentType} = lastField;
+  switch (componentType) {
+    case 'menu':
+      return <BulletPointMenuButton {...lastField} />;
+    case 'button-link':
+      return <ButtonLink {...lastField} />;
+  }
+};
+
+const buildNestedRow = (row: NestedRow) => {
+  const {componentType} = row;
+  switch (componentType) {
+    case 'errors-table':
+      return <ErrorsTable {...row} />;
+    case 'expandible-errors-table':
+      return <ActionableExpandableErrorsTable {...row} />;
+  }
+};
+
+const ActionableExpandableErrorsTable = (props: Props, legacyContext: WebContextValues) => {
+  const {
+    columns,
+    rows = [],
+    lastField,
+    stickyFirstColumn = false,
+    stickyLastColumn = false,
+    ariaDescribedby,
+    columnWidth = `${(1 / columns.length) * 100}%`,
+    isNestedTable = false
+  } = props;
+  const translate = GetTranslateFromContext(legacyContext);
+
+  /**
+   * State variable to keep track of all the expanded rows
+   * By default, nothing expanded. Hence initialized with empty array.
+   */
+  const [expandedRows, setExpandedRows] = useState<number[]>([]);
+
+  /**
+   * State variable to keep track which row is currently expanded.
+   */
+  const [expandState, setExpandState] = useState<ExpandState>({});
+
+  /**
+   * This function gets called when show/hide link is clicked.
+   */
+  const handleExpandRow = (index: number) => () => {
+    const isRowExpanded = expandedRows.includes(index);
```

Comment:
includes de Array n'est supporté par tous les navigateurs

```suggestion
    const isRowExpanded = _.includes(expandedRows, index);
```